bindings/cs/rl.net/LiveModel.cs

```diff
@@ -71,6 +77,24 @@ public bool TryChooseRank(string actionId, string contextJson, RankingResponse r
             return result == NativeMethods.SuccessStatus;
         }
 
+        public bool TryChooseRank(string actionId, string contextJson, ActionFlags flags, out RankingResponse response, ApiStatus apiStatus = null)
+        {
+            response = new RankingResponse();
+            return this.TryChooseRank(actionId, contextJson, flags, response, apiStatus);
+        }
+
+        public bool TryChooseRank(string actionId, string contextJson, ActionFlags flags, RankingResponse response, ApiStatus apiStatus = null)
```

Comment:
Do we need this method as public? It seems there is no need for a caller to create RankingResponse object and pass it as parameter.

Reply:
I am ok to remove it, but for now I was just following the previously introduced style (see methods above)

Reply:
I see